network/server/tchannelthrift/thrift/rpc.thrift

```diff
@@ -50,7 +50,7 @@ service Node {
 	void write(1: WriteRequest req) throws (1: WriteError err)
 	void writeBatch(1: WriteBatchRequest req) throws (1: WriteBatchErrors err)
 	FetchResult fetch(1: FetchRequest req) throws (1: Error err)
-	FetchRawBatchResult fetchRawBatch(1: FetchRawBatchRequest req) throws (1: Error err)
+	FetchRawBatchResult fetchRawBatch(1: FetchRawBatchRequest req)
```

Comment:
We now return errors inside the batch result similar to the write batch.  I could also change write batch to return the errors as a result to match the pattern we're using here now.